### PR TITLE
Web Scraping in Python

### LINKED ISSUE #8007: Web Scraping in Python
### Description

I would like to contribute with documentation on Web Scraping in Python in which I'll cover how to process HTML documents and HTTP requests &  techniques for selecting and extracting data from websites.

### Domain

Python_Domain

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Vishaka830**
<hr>

## Description 📜

I've completed the documentation of Web Scraping in Python Language.
Fixes #8007

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [X] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [X] Python_Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Vishaka830** [comment]
@adityabisoi Sir please have a look and let me know if any changes are required.

File: Python/web_scraping_in_python/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+I've completed the documentation of web scraping in Python that consists of:
+<ul>
```
Comment (apeksha235) [inline Python/web_scraping_in_python/README.md:2]: avoid using html tags in md file, please use md formatting.

File: Python/web_scraping_in_python/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  </li>
+  <li><b>Open Source Programming Language</b>
+Python has tremendous help from the community since it is an open-source programming language.
+  </li>
+  <li><b>Wide scope of Applications</b>
+Python can be utilized for different programming assignments going from little shell contents to big business web applications.</li>
+<br/><br/></li>
+ </ul>
+ 
+ <li><b>Python Libraries Used For Web Scraping</b><br/><br/>
```
Comment (apeksha235) [inline Python/web_scraping_in_python/README.md:33]: mention and explain other libraries as well that are used for web scraping

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@Vishaka830 please make these changes

File: Python/web_scraping_in_python/README.md (hunk 2 above)
Comment (Vishaka830) [reply Python/web_scraping_in_python/README.md]: @apeksha235  I've made all the changes that you specified. Please have a look

Comment (Vishaka830) [reply Python/web_scraping_in_python/README.md]: @apeksha235 @adityabisoi please review

**Comment by Vishaka830** [comment]
@adityabisoi @apeksha235 Please review. This is my 5th PR and I really want it to be merged.

**Pull request merged**